### DISCUSSION TITLE
Conversion to TypeScript

### DISCUSSION THREAD

**Discussion by seanvarnham** [vaarvik/wp-gb#3] [Ideas]
Hey, in the past I've relied on your site to work with the components, and I've found it more useful that the actual dev docs from WP at times. I'd like to help out in the future, but one suggestion I have is to introduce TS. Is this something you're familiar with?

**Comment by vaarvik** [comment]
> Hey, in the past I've relied on your site to work with the components, and I've found it more useful that the actual dev docs from WP at times. I'd like to help out in the future, but one suggestion I have is to introduce TS. Is this something you're familiar with?

Hi! Very glad to hear that the site is helpful. I do for sure think it should be converted to TypeScript. I've also been thinking about using Next.js lately to make it more SEO friendly.
